### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ddl-ahmet-gyger**
Updating licenses information

### CODE REVIEW

**Comment by ddl-ahmet-gyger** [comment]
@niole - can you review?

**Comment by ddl-ahmet-gyger** [comment]
https://dominodatalab.atlassian.net/browse/DOM-53273

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # OSS LLM Inference Template
 
 ## License
-This template is licensed to Customer subject to the terms of the license agreement between Domino and the Customer on file.
+This template is licensed under Apache 2.0 and contains the following open source components: Transformer (Apache 2.0)[https://github.com/huggingface/transformers/blob/main/LICENSE], Falcon (Apache 2.0)[https://huggingface.co/tiiuae/falcon-7b/blob/main/README.md], MLFlow (Apache 2.0)[https://github.com/mlflow/mlflow/blob/master/LICENSE.txt], NVIDIA [EULA](https://docs.nvidia.com/cuda/eula/index.html#license-grant)).
```
Comment (niole) [inline README.md]: the parenthesis don't look balanced, the urls maybe aren't rendering correctly: 
Screen Shot 2023-12-20 at 1 25 12 PM

**Comment by ddl-ahmet-gyger** [comment]
Thanks @niole, good catch - I've fixed that.

**Review by niole — APPROVED** [review]

**Pull request merged**